pegasus/sites.v3/code.org/styles/050-theme.css

```diff
@@ -131,10 +131,50 @@ body {
 /* styling for the embedded dashboard components... */
 
 #user_hero #welcome {
-  margin-top:10px;
```

Comment:
does this css also need to be in dashboard? if so, would it make sense to put it in shared?